content/docs/tasks/traffic-management/app-health-check/index.md

```diff
@@ -114,27 +128,111 @@ Remove the mutual TLS policy and corresponding destination rule added in the ste
 
 ## Liveness and readiness probes with HTTP request option
 
-This section shows how to configure health checking with the HTTP request option.
+This section shows how to configure health checking with the HTTP request
+option. Firstly it discusses the case when the service is using same port
+for regular traffic and health checking. Then it discusses the case with
+different port.
```

Comment:
Consider the following edits to enhance clarity:
This section shows how to configure health checking for the HTTP request
option:
1. We cover when the service uses the same port for regular traffic and for health checking without and with mTLS enabled.
1. We cover when the service uses one port for regular traffic and another for health checking without and with mTLS enabled.